elements/rh-tabs/test/rh-tabs.spec.ts

```diff
@@ -0,0 +1,165 @@
+      expect(buttonBeforeStyles).to.be.equal(tokens.get('--rh-border-width-sm'));
+    });
+  });
+
+  describe('inset tabs', function() {
+    it('should have inset styles', async function() {
+      element.setAttribute('box', 'inset');
+      await nextFrame();
+      const insetEl = element.shadowRoot!.querySelector(`[part="tabs"]`)!;
+      const insetStyles = getComputedStyle(insetEl).marginInline;
+      expect(insetStyles).to.be.equal(tokens.get(`--rh-space-2xl`));
+    });
+  });
+
+  describe('keyboard navigation', function() {
+    let firstItem: RhTab;
+    let secondItem: RhTab;
+    let lastItem: RhTab;
+    let initialActiveElement: Element|null;
+
+    beforeEach(async function() {
+      [firstItem, secondItem] = element.querySelectorAll<RhTab>('rh-tab');
+      lastItem = [...element.querySelectorAll<RhTab>('rh-tab')].pop();
+      await sendKeys({ press: 'Tab' });
+      initialActiveElement = document.activeElement;
+    });
+
+    it('should focus a this first jump-links-item', function() {
+      expect(document.activeElement).to.equal(firstItem);
+    });
+
+    describe('pressing right arrow key', function() {
+      beforeEach(async function() {
+        await sendKeys({ press: 'ArrowRight' });
+        await allUpdates(element);
+        await nextFrame();
+      });
+
+      it('should focus a rh-tab button', function() {
+        expect(document.activeElement).to.be.an.instanceof(RhTab);
+      });
+
+      it('should change focus when keyboard navigation is used', function() {
+        expect(document.activeElement).to.not.equal(initialActiveElement);
+      });
+
+      it('should focus the second rh-tab item', function() {
+        expect(document.activeElement).to.equal(secondItem);
+      });
+    });
+
+    describe('pressing left arrow key', function() {
+      beforeEach(async function() {
+        await sendKeys({ press: 'ArrowLeft' });
+        await allUpdates(element);
+        await nextFrame();
+      });
+
+      it('should focus a rh-tab button', function() {
+        expect(document.activeElement).to.be.an.instanceof(RhTab);
+      });
+
+      it('should change focus when keyboard navigation is used', function() {
+        expect(document.activeElement).to.not.equal(initialActiveElement);
+      });
+
+      it('should focus the last rh-tab item', function() {
+        expect(document.activeElement).to.equal(lastItem);
+      });
+    });
+  });
+
+  describe('on small screen', function() {
+    beforeEach(async function() {
+      await setViewport({ width: 320, height: 640 });
+      await allUpdates(element);
+    });
+
+    it('should overflow if too wide', async function() {
+      const tabs = element.shadowRoot!.querySelector('[part="tabs"]')!;
+      const tabsOverflow = getComputedStyle(tabs).overflowX === 'auto';
+      expect(tabsOverflow).to.be.equal(true);
+    });
+
+    it('should have visible scroll buttons if overflowed', async function() {
+      await aTimeout(150);
```

Comment:
can we remove this?

Reply:
removed in [207d60b](https://github.com/RedHat-UX/red-hat-design-system/pull/634/commits/207d60b5aa48f6ea202a8675b5f73d93c19e0b19)